<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Martin Koller</author>
 <class>ConfigureDesktopDialog</class>
 <widget class="QDialog" name="ConfigureDesktopDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Wallpaper</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Background Color</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KColorCombo" name="kcolorcombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Wallpaper File</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KUrlRequester" name="kurlrequester"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="iconView">
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Scale Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="origSizeButton">
        <property name="text">
         <string>Or&amp;iginal Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scaledIgnoreRatioButton">
        <property name="text">
         <string>Sca&amp;led Ignoring Aspect Ratio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scaledKeepRatioButton">
        <property name="text">
         <string>Scale&amp;d Keep Aspect Ratio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scaledKeepRatioClipButton">
        <property name="text">
         <string>Scaled Keep Aspec&amp;t Ratio and Clip</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureDesktopDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureDesktopDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
